test/blueprints/Do.Test.Blueprints.Service.Application/RestApi/Analyzer/Entity.generated.cs

```diff
@@ -13,13 +13,34 @@ public class EntityController
     public EntityController(IServiceProvider serviceProvider) =>
         _serviceProvider = serviceProvider;
 
+    public record ByRequest(
+        Guid Guid = default,
+        string String = default,
+        string StringData = default,
+        int Int32 = default,
+        Uri Uri = default,
+        object Dynamic = default,
+        Status Status = default,
+        DateTime DateTime = default
```

Comment:
these appear `PascalCase` in swagger, should be `camelCase`.  making them camel case in a record would conflict with stylecop. use an attribute to override this, or move them to method separately